docs/How  to use the epos plugin.md

```diff
@@ -0,0 +1,158 @@
+# How to use the Epos plugin of the FAIR EVA 
+To use the FAIR EVA you need access to a terminal 
+
+## 1.Follow the steps indicated in the  documentation
```

Comment:
To me, this heading should be something like `## 1. Deploy FAIR EVA` and then under it:
`Follow the steps indicated in the [documentation](..)`

Reply:
Changed